### DISCUSSION TITLE
We are distracting ourselves. Don't do the same to them

### DISCUSSION THREAD

**Discussion by ghost** [rms-support-letter/rms-support-letter.github.io#5099] [General]
> "An eye for an eye will leave the whole world blind.”

Some of them are creating extensions to discriminate us, and they are talking shit about us in Twitter. It's a very ugly. But, Let's not lower our standards by indulging in THOSE SAME activities. Harrassing the people who harrass is NOT going to do any favour to us, or the FSF, or the struggle we are facing by the corporates to fade away the free software movement. We should be the good guys AND need to focus our efforts on the smear campaign against RMS. We SHOULD NOT be known for these cheap activities.

Regards,
Nainar

**Comment by appetrosyan** [comment]
I second this. And I'd also like to add that harassment is giving them extra ammunition. So far, their guilt and complicity in a crime is obvious through various coercions. If we do the same, we can no longer claim that they are the **only** ones violating the guidelines of GitHub, GNU Kind Communication etc. 

We are civil and we must indulge in a civil conversation. You are allowed to point out facts, as long as you are being respectful, and the facts are supported by evidence. Do not stoop to their level. 

If you want to single them out, move all your projects to GPLv3 only. **You** are allowed to update it to a later version as the author, but that will prevent them from relaxing the GPL constraints on your work.

**Comment by feverzsj** [comment]
It's far from harassment (like what they are doing right now). We just help them to be better coders, (though not necessary better human). And we also need to stand out to protect our signees from the real harassment.

**Comment by appetrosyan** [comment]
> It's far from harassment (like what they are doing right now). We just help them to be better coders, (though not necessary better human). And we also need to stand out to protect our signees from the real harassment.

I think this is superfluous. We should register their mistakes somewhere, but don't necessarily serve them a way to better harass us on a silver platter.

**Comment by N0rbert** [comment]
We know the name of harassing person, it is `aaronbassett` as per https://www.opennet.ru/opennews/art.shtml?num=54870 . We may want to send abuses against it.

**Comment by pda0** [comment]
I have already said that we ourselves can use the that tools to see each other. There is something beautiful about that, using a tool designed to hate to make friends. 😉
